idna/tests/uts46.rs

```diff
@@ -10,6 +10,9 @@ use std::char;
 use idna::uts46;
 use test::TestFn;
 
+const SKIP_TEST: &'static [&'static str] = &["0A.\\u05D0", "0a.\\u05D0", "0a.xn--4db"
+    ,"1.걾6.𐱁\\u06D0", "1.걾6.𐱁\\u06D0","1.xn--6-945e.xn--glb1794k"];
```

Comment:
Probably explaining why these are skipped in a comment would be nice ‒ maybe linking to the issue?